frontend/resourceadm/components/RightTranslationBar/RightTranslationBar.tsx

```diff
@@ -32,93 +31,73 @@ export type RightTranslationBarProps = {
-        } else {
-          e.preventDefault();
-          // Tab was pressed without Shift, perform custom behavior (onLeaveLastField)
-          onLeaveLastField(e);
-        }
-      }
-    };
+  const displayNField = (lang: 'nn' | 'en') => {
+    const label = `${title} (${lang === 'en' ? t('language.en') : t('language.nn')})`;
 
-    const displayNField = (lang: 'nn' | 'en', isLast: boolean) => {
-      const label = `${title} (${lang === 'en' ? t('language.en') : t('language.nn')})`;
-
-      if (usesTextArea) {
-        return (
-          <Textarea
-            value={value[lang]}
-            onChange={(e) => handleChange(lang, e.currentTarget.value)}
-            rows={5}
-            label={label}
-            error={showErrors && value[lang] === ''}
-            ref={!isLast ? (ref as React.Ref<HTMLTextAreaElement>) : undefined}
-            onKeyDown={(e) => (isLast ? handleTabOutOfTranslationBar(e) : undefined)}
-            onBlur={onBlur}
-            size='small'
-          />
-        );
-      }
+    if (usesTextArea) {
       return (
-        <Textfield
+        <Textarea
           value={value[lang]}
-          onChange={(e) => handleChange(lang, e.target.value)}
+          onChange={(e) => handleChange(lang, e.currentTarget.value)}
+          rows={5}
           label={label}
           error={showErrors && value[lang] === ''}
-          ref={!isLast ? (ref as React.Ref<HTMLInputElement>) : undefined}
-          onKeyDown={(e) => (isLast ? handleTabOutOfTranslationBar(e) : undefined)}
           onBlur={onBlur}
           size='small'
         />
       );
-    };
-
+    }
     return (
-      <div className={classes.wrapper}>
-        <div className={classes.topWrapper}>
-          <GlobeIcon
-            title={t('resourceadm.right_translation_bar_translation')}
-            fontSize='1.5rem'
-            className={classes.icon}
-          />
-          <Heading size='xsmall' level={2} className={classes.topText}>
-            {t('resourceadm.right_translation_bar_title')}
-          </Heading>
-        </div>
-        <div className={classes.bodyWrapper}>
-          <Alert severity='info'>
-            <Paragraph size='small'>{t('resourceadm.right_translation_bar_alert')}</Paragraph>
-          </Alert>
-          <div className={classes.inputWrapper}>{displayNField('nn', false)}</div>
-          <div className={classes.inputWrapper}>{displayNField('en', true)}</div>
-        </div>
-      </div>
+      <Textfield
+        value={value[lang]}
+        onChange={(e) => handleChange(lang, e.target.value)}
+        label={label}
+        error={showErrors && value[lang] === ''}
+        onBlur={onBlur}
+        size='small'
+      />
     );
-  },
-);
+  };
+
+  return (
+    <div className={classes.wrapper}>
+      <div className={classes.topWrapper}>
+        <GlobeIcon
+          title={t('resourceadm.right_translation_bar_translation')}
+          fontSize='1.5rem'
```

Comment:
Can we move the ```fontSize``` to the CSS? 
I know this is not changed in this current branch, but I think we can clean it up as we are here 😄